Source/DafnyTestGeneration/Main.cs

```diff
@@ -110,9 +110,9 @@ public static class Main {
         yield break;
       }
       var dafnyInfo = new DafnyInfo(program);
-      var rawName = sourceFile.Split("/").Last().Split(".").First();
+      var rawName = sourceFile.Split(new[] { '/', '\\' }).Last().Split(".").First();
 
-      yield return $"include \"{sourceFile}\"";
+      yield return $"include \"{sourceFile.Replace("\\", "\\\\")}\"";
```

Comment:
It outputs an include statement that will include the Dafny generated files, but since in Windows the paths are backslashes, we cannot output raw backslashes in Dafny code, they need to be escaped.

Reply:
I remember someone suggesting to use verbatim strings for code like this :D

Consider moving the replace into a method `EscapeDafnyStringLiteral`

Reply:
The waterer watered :-P
Considered and done.